scripts/tools/zap/tests/outputs/all-clusters-app/app-templates/access.h

```diff
@@ -49,6 +49,8 @@
     0x0000003E, /* Cluster: Operational Credentials, Attribute: NOCs, Privilege: administer */ \
     /* Cluster: Group Key Management, Attribute: GroupKeyMap, Privilege: view */ \
     /* Cluster: User Label, Attribute: LabelList, Privilege: view */ \
+    0x00000046, /* Cluster: ICD Management, Attribute: RegisteredClients, Privilege: administer */ \
```

Comment:
why these changes?

Reply:
I was using the wrong Zap configuration, fixed.